odl/discr/discretization.py

```diff
@@ -400,163 +441,34 @@ def interpolation(self):
 
         See Also
         --------
-        DiscretizedSet.interpolation : For full description
+        DiscretizedSpace.interpolation : For full description
         """
-        return self.space.interpolation(self.ntuple)
+        return self.space.interpolation(self.tensor)
+
+    def __ipow__(self, p):
+        """Implement ``self **= p``."""
+        # Falls back to `LinearSpaceElement.__ipow__` if `self.tensor`
+        # has no own `__ipow__`. The fallback only works for integer `p`.
+        # TODO: do we even need this??
```

Comment:
A quick check reveals that this is actually needed. The thing is that `DiscretizedSpaceElement` inherits from `Tensor` (the abstract one) which has no own `__ipow__` and thus falls back to the `LinearSpaceElement` version which only works for integers.
Others are not needed since `__pow__` uses `__ipow__` and the rest has no modification in `Tensor` classes either.